pages/pipelines/trigger_step.md.erb

```diff
@@ -17,78 +17,96 @@ _Required attributes:_
   <tr>
     <td><code>trigger</code></td>
     <td>
-      The slug of the triggered pipeline. The pipeline slug must be lowercase.<br>
+      The slug of the pipeline to create a build. The pipeline slug must be lowercase.<br>
       <em>Example:</em> <code>"another-pipeline"</code>
     </td>
   </tr>
 </table>
 
-Optional attributes:
+_Optional attributes:_
 
 <table>
   <tr>
     <td><code>build</code></td>
     <td>
-      TODO
+      A map of attributes for the triggered build. See below for the list of attributes.
```

Comment:
Perhaps we just say "An optional map of attributes for the triggered build." and people would know to look down? I used "map" because we use that for these types of things.